### DISCUSSION TITLE
Welcome to confetti Discussions!

### DISCUSSION THREAD

**Discussion by reindert-vetter** [confetti-framework/confetti#95] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by marcelloh** [comment]
Marcel (Senior Software Architect & Go trainer) (ex. colleague of Bart H.) is waiting for an example
or a step-by-step "how to" (on dev.to ?) and then some promotion on reddit perhaps to that explanation.

**Reply by reindert-vetter** [reply]
At https://www.confetti-framework.com/docs/get-started/your-first-project you can see how you can create a page and an API endpoint in a few steps. What things are you still missing there? Do you just need more examples with validation, more routers and, for example, writing to logs?

I will share it on dev.to and then on Reddit. That is a good idea!

**Reply by marcelloh** [reply]
ok, I missed that one. Perhaps a todo application with some more actions? I can do that and share my thoughts?

**Reply by reindert-vetter** [reply]
@marcelloh That is a great idea! If you do, I can link to that at the bottom of that page.

**Comment by Issa-projects** [comment]
Hello there!
I do not see the example app repo has any use of the framework. Is this meant to be like that? I thought the example will be at least using the framework to show how it works.

**Reply by reindert-vetter** [reply]
This project is being renewed. A stable version will be released in August. It is better to wait until then.
I am now working on rewriting everything and connecting everything to the standard go functionality.

To use the old version you can go to:
https://github.com/confetti-framework/confetti/tree/1e5209f7a189ba2551f5b4284b26b3c12e60613c
